### PR TITLE
Update environment-variable-expose-pod-information.md

### PR DESCRIPTION
**Pull request by guzj11**
fix the broken url.

### CODE REVIEW

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

File: content/zh/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Together, these two ways of exposing Pod and Container fields are called the
 有两种方式可以将 Pod 和 Container 字段呈现给运行中的容器：
 
 * 环境变量
-* [卷文件](/docs/resources-reference/{{< param "version" >}}/#downwardapivolumefile-v1-core)
+* [卷文件](/zh/docs/resources-reference/{{< param "version" >}}/#downwardapivolumefile-v1-core)
```
Comment (tengqm) [inline content/zh/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md:42]: In English, `/docs/resources-reference` is a redirection. We don't have such redirection for Chinese. So this revision will not work. The link is still a 404.

The correct link target is now `/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api`.

Comment (guzj11) [reply content/zh/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md]: thank you for suggestion,
suggestion applied.

**Pull request merged**